README.md

```diff
@@ -27,34 +27,48 @@ The third-party dependencies themselves are described, each in its own Python fi
 
 To modify some dependency (e.g. glog):
 * Look at the corresponding python file in build_definitions, e.g. https://github.com/yugabyte/yugabyte-db-thirdparty/blob/master/python/build_definitions/glog.py, to see where it is pulling the dependency from. Usually it is a GitHub repo, either the open-source upstream repo or our own fork.
-* In case of glog, it is https://github.com/yugabyte/glog. Fork that repo on GitHub, to e.g. https://github.com/yourgithubusername/glog. Make all the necessary changes in your fork, push changes to a branch on your fork, and create a tag, e.g. `0.4.0-yb-yourusername-1`. Push the tag to your fork.
-* Modify the dependency python file, i.e. glog.py in this case, to reference your own repo and tag:
-
+* In case of glog, it is https://github.com/yugabyte/glog. Fork that repo on GitHub, to e.g. https://github.com/yourgithubusername/glog. Clone your fork of the repo and make all the necessary changes in your local checkout.
+* To test locally, pass `local_archive` pointing to your clone path (`local_archive` is only meant for testing and should not be submitted in your PR):
+```python
+        super(GLogDependency, self).__init__(
+            name='glog',
+            version='0.4.0-yb-1',
+            url_pattern='https://github.com/yugabyte/glog/archive/v{0}.tar.gz',
+            local_archive='~/path/to/glog',
```

Comment:
Does this really understand `~` as the home directory? (In order to do that, you will need `os.path.expanduser`).

Reply:
I tested with an absolute path. I'll change the README.md to reflect an absolute path.